inyoka/wiki/macros.py

```diff
@@ -217,7 +217,7 @@ def build_node(self, context, format):
 
 class FilterByMetaData(macros.Macro):
     """
-    Filter pages by their metadata
+    Filter pages by their metadata.
```

Comment:
(Short) Description for the `NOT` and `EXACT` keywords in filters would be beneficial.